addons/Todo_Manager/Dock.gd

```diff
@@ -41,6 +42,11 @@ func build_tree() -> void:
 	var root := tree.create_item()
 	root.set_text(0, "Scripts")
 	for todo_item in todo_items:
+		var ignore := false;
+		for ignore_path in ignore_paths:
+			if todo_item.script_path.begins_with("res:///" + ignore_path):
+				ignore = true;
+		if ignore: continue;
```

Comment:
I'm not a huge fan of one-liners like this. Could it be changed to:
```
if ignore:
    continue
```

Reply:
Fixed and pushed the change